src/schoology-extractor/tests/mapping/test_assignments.py

```diff
@@ -26,7 +26,7 @@ def result() -> pd.DataFrame:
             "title": "The title",
             "type": "assignment",
             "description": "The description",
-            "due": "8/21/2020  11:59:00 PM",
+            "due": "2020-8-21  23:59:00",

```

Comment:
The actual API results have a single space between the date and time. Does that matter? Will it be ok for this test value to have two spaces?

Reply:
fixed